sdk/javascript/package.json

```diff
@@ -1,7 +1,10 @@
 {
   "name": "sawtooth-sdk",
   "version": "0.0.0",
-  "description": "An SDK for interacting with the Sawtooth Lake distributed ledger",
+  "description": "An SDK for interacting with the Hyperledger Sawtooth distributed ledger.",
+  "keywords": ["hyperledger", "blockchain", "signing", "crypto", "protobuf"],
+  "homepage": "http://intelledger.github.io/",
```

Comment:
Technically, that's the documentation site.  The home page is https://www.hyperledger.org/projects/sawtooth, I think